airbyte-webapp/src/packages/cloud/views/users/AccountSettingsView/components/EmailSection.tsx

```diff
@@ -51,17 +51,23 @@ export const EmailSection: React.FC = () => {
     void,
     Error,
     { email: string; passwd: string }
-  >(async ({ email, passwd }) => {
-    await authService.changeEmail(email, passwd);
-    return userService.changeEmail(email);
-  });
+  >(
+    async ({ email, passwd }) => {
+      await authService.changeEmail(email, passwd);
```

Comment:
Logic leak here. Let's place it into `AuthService` just how all other auth services handle it. It already has access to authService and userService, so you won't need to do any initial initialization